site/public/css/server.css

```diff
@@ -1061,7 +1061,7 @@ div.full_height {
     margin-top: 5px;
 }
 
-.cat-buttons:hover {
+.cat-buttons:hover, .cat-buttons:focus{
     opacity: 0.8;
```

Comment:
This feature works but the difference is very hard to notice when hovering/focusing on a category button.